charts/common/templates/_drax.tpl

```diff
@@ -66,3 +66,29 @@ The Role this component serves in the overall application
 
 {{- ($values.drax).role | default ($.Values.drax).role -}}
 {{- end -}}
+
+
+{{/*
+The Accelleran Product set for the deployment
+*/}}
+{{- define "accelleran.common.drax.accelleranProduct" -}}
```

Comment:
I'm not sure to what extend it's really useful to have this in the common charts (already)... There is only one level where this check makes sense I guess and that would be within the dashboard (for now at least). So I would keep it there as long as we don't need any further differentiation.